packages/builder/src/components/userInterface/AppPreview/iframeTemplate.html

```diff
@@ -1,6 +1,7 @@
 <html>
   <head>
     <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
```

Comment:
Should we import the CSS for remix in the icon component itself? That way it will get bundled with the client apps.

Reply:
I think that would be better, struggled a lot with sorting it out though, got it working in the builder but then I had issues in the actual preview and generated apps. Looked like a pathing issue. Open to suggestions and/or help on how to do it. 🙏